app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= stylesheet_link_tag "application"%>
+
+<div class="logger_in_form">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field custom-login">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field custom-login">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <div class="actions">
+  <button class="mybtn" type="button"><%= f.submit "Log in" %><button>
+  </div>
+<% end %>
```

Comment:
Wireframe
![image](https://github.com/obibaadoma/Blog_App/assets/78251927/ea6c5b94-df17-4f41-8c10-68267c86bfe7)

Your design
![image](https://github.com/obibaadoma/Blog_App/assets/78251927/c7f55ad6-50c8-483f-af63-a219e393236d)


According to the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md), The views in devise for the registration and login should be modified to match the wireframes and their styling.

For a better UX design and to match the wireframe. , I suggest that we remove the inner border of the button as indicated above (both `Sign up` and `Login` page)😊